spec/syntax.md

```diff
@@ -97,6 +97,14 @@ Attempting to parse a _message_ that is not _valid_ will result in a _Data Model
 
 A **_<dfn>message</dfn>_** is the complete template for a specific message formatting request.
 
+
+A **_<dfn>variable</dfn>_** is a data value bound to a _name_.
```

Comment:
I don't think "name" has been defined -- or is it meant to refer to _name_ as defined in the grammar? But if so, then _variable_ doesn't need to be defined here either, since it's in the grammar.

Also, this conflicts with the later statement that: "A local-declaration binds a variable to the resolved value of an expression." Maybe that sentence should be changed to say "name" instead of "variable"?

Reply:
> I don't think "name" has been defined

`name` has a definition [here](https://github.com/unicode-org/message-format-wg/blob/main/spec/syntax.md#names-and-identifiers)

_variable_ needs to be defined because we (try to) link it many times in the spec.

> Maybe that sentence should be changed to say "name" instead of "variable"?

Good point. Perhaps by saying something like:

> A `local-declaration` creates a _variable_ by binding a _name_ to the resolve value of an _expression_.